<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScannerDialog</class>
 <widget class="QDialog" name="ScannerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="scannerSearch">
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>60</y>
         <width>211</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Scanner search</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="scannerList">
      <widget class="QComboBox" name="scannerComboBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>100</y>
         <width>111</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="scanDialog">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="scannerLabel">
           <property name="text">
            <string>Scanner:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="scannerName">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="resolutionLabel">
           <property name="text">
            <string>Resolution</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="resolutionComboBox"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="scanModeLabel">
           <property name="text">
            <string>Scan mode</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="scanModeComboBox"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ScannerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ScannerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
